Apps/WebClient/src/ClientApp/src/components/dependent/tabs/DependentDashboardTabComponent.vue

```diff
@@ -1,20 +1,277 @@
+        });
+    }
+
+    private handleFederalProofOfVaccinationDownload(): void {
+        this.logger.debug(`Handle federal proof of vaccination download`);
+        SnowPlow.trackEvent({
+            action: "click_button",
+            text: "Dependent_Proof",
+        });
+        this.retrieveAuthenticatedVaccineRecord({
+            hdid: this.dependent.ownerId,
+        });
+    }
+
+    private showSensitiveDocumentDownloadModal(): void {
+        this.sensitiveDocumentDownloadModal.showModal();
+    }
 }
 </script>
 <template>
-    <div>Dashboard Tab Content</div>
+    <div>
+        <LoadingComponent
+            :is-loading="isVaccineRecordDownloading"
+            :text="vaccineRecordStatusMessage"
+        />
+        <b-row cols="1" cols-lg="2" cols-xl="3">
+            <b-col v-for="entry in entryTypes" :key="entry.type" class="p-3">
+                <hg-card-button
+                    :title="entry.name"
+                    has-chevron
+                    :data-testid="`dependent-entry-type-${dependent.ownerId}`"
+                    @click="handleClickEntryType(entry.type)"
+                >
+                    <template #icon>
+                        <hg-icon
+                            :icon="entry.icon"
+                            class="entry-link-card-icon align-self-center"
+                            size="large"
+                            square
+                        />
+                    </template>
+                </hg-card-button>
+            </b-col>
+            <b-col v-if="showFederalProofOfVaccination" class="p-3">
+                <hg-card-button
+                    title="Proof of Vaccination"
+                    :data-testid="`proof-vaccination-card-btn-${dependent.ownerId}`"
+                    @click="showSensitiveDocumentDownloadModal()"
+                >
+                    <template #icon>
+                        <hg-icon
+                            icon="download"
+                            class="entry-link-card-icon align-self-center"
+                            size="large"
+                            square
+                        />
+                    </template>
+                </hg-card-button>
+            </b-col>
+        </b-row>
+        <MessageModalComponent
+            ref="sensitivedocumentDownloadModal"
+            title="Sensitive Document Download"
+            message="The file that you are downloading contains personal information. If you are on a public computer, please ensure that the file is deleted before you log off."
+            @submit="handleFederalProofOfVaccinationDownload"
+        />
+        <MessageModalComponent
+            ref="vaccineRecordResultModal"
+            ok-only
+            title="Alert"
+            :message="vaccineRecordResultMessage"
+        />
+    </div>
 </template>
+
+<style lang="scss" scoped>
+@import "@/assets/scss/_variables.scss";
+
+.canada-government-logo {
+    height: 1.5em;
+}
```

Comment:
This class isn't needed in this component.

Reply:
Changed.